elasticsearch-jmeter/azuredeploy.json

```diff
@@ -0,0 +1,426 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username used when provisioning virtual machines"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password used when provisioning virtual machines"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "ResourceGroup",
+                "West US",
+                "East US",
+                "East US 2",
+                "Central US",
+                "South Central US",
+                "North Central US",
+                "East Asia",
+                "Southeast Asia",
+                "West Europe",
+                "North Europe"
+            ],
+            "defaultValue": "ResourceGroup",
+            "metadata": {
+                "description": "Location where resources will be provisioned"
+            }
+        },
+        "virtualNetworkName": {
```

Comment:
Let's prefix this with existing to make it clear this needs to be setup beforehand just in case users don't read the README first